include/libirc/connectivity.h

```diff
@@ -1054,10 +1162,83 @@ dihedrals(const Matrix& distance_m,
     }
   }
 
+  // TODO Check if enough coordinates found elsewhere
+  
   // Return list of dihedral angles
   return dih;
 }
 
```

Comment:
@peterbygrave Can you add some documentation (or reference) here as well please?